storage/rocksdb/myrocks_hotbackup.py

```diff
@@ -1,4 +1,4 @@
-#!@PYTHON_SHEBANG@
```

Comment:
why are you changing this? The change is the default unless overwritten by RPM/Deb builds.

Reply:
Because of the problems with bytes and str, I suspect that my modified script cannot be run with python2